app/views/users/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="container">
+<div>
+<div class="image">Images</div>
+<div class="card">
+<h1 class="header"><%=@user.name%></h1>
+<p class="footer">Number of posts: <%=@user.posts_counter %></p>
```

Comment:
- [ ] Good that you are displaying the user name and user post counter :+1: But please make sure you display the user photo as well using `<img src="<%= user.photo%>" alt="<%= user.name%>">` :pray: